core_impl/src/main/java/com/google/instrumentation/internal/logging/Logger.java

```diff
@@ -0,0 +1,28 @@
+/*
```

Comment:
I understand after reading further - log on errors. I think we can do away with this for now, since we have defined error-less operations for v1, and even when we do have them, we will (probably) want to use exceptions instead.